packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -289,18 +289,6 @@ export class Pipeline extends PipelineBase {
     return this.stages.length;
   }
 
-  /**
-   * Returns all of the {@link CrossRegionSupportStack}s that were generated automatically
-   * when dealing with Actions that reside in a different region than the Pipeline itself.
-   */
-  public get crossRegionSupport(): { [region: string]: CrossRegionSupport } {
```

Comment:
Let’s expose this with an @experimental tag

Reply:
Done.